client/app/recordings/new/route.js

```diff
@@ -7,11 +7,62 @@ export default Route.extend({
     this.container = options.container;
   },
 
-  render() {
-    this.view = new View({
-      model: new Model(),
-      editing: true
-    });
-    this.container.show(this.view);
-  }
+  render(params) {
+    var that = this;
+    function renderPage(_model){
+      that.view = new View({
+        model: _model,
+      });
+      that.container.show(that.view);
+    }
+    function retrieveListAndRender(_model){
+      var listxhr = new XMLHttpRequest();
```

Comment:
In Backbone interaction with the API should be mediated through models and collections. I would suggest rewriting this logic as a collection. In fact, there already is a collection that fetches organizations. There is in fact another layer of abstraction on top of collections ('stores') that cache the data. Look at the organizations router for an example of how to fetch organizations.